GameLogic/player_management/Equipment.cs

```diff
@@ -8,16 +9,32 @@ public enum WeaponType
 }
 
 public class Helmet :Item{
+
+    public Helmet ( string name, int price, int def, double defPerc) : base(name, price)
+    {
+        Def = def;
+        DefPerc = defPerc;
+    }
```

Comment:
Can't this be added to an abstract class? You write it everywhere.

Reply:
Nah, cause I have class Armour(I think about renaming it to armourSet) and it must contain three pieces of Set, each with specified type: helmet, chest plate and leggins, 
Refactoring will cause some problems checking two items of armor of the same type in a set (for example, two helmets)